ex-1.sc

```diff
@@ -0,0 +1,20 @@
+def examWords(args: List[String]): Unit = {
+  val filteredArgs = args.filter(_.length > 2)
+  if (filteredArgs.nonEmpty) {
+    val maxWord = filteredArgs.maxBy(_.length)
+    val minWord = filteredArgs.minBy(_.length)
+
+    print("Longer word/s: ")
```

Comment:
I would separate in at least 2 functions.
One that is a pure function that has no side effects, that accepts the list and returns the result of the computation and one other function that print the result.
Doing that has the main advantage is
The code is more testable. At the moment the code is not testable because the `examWords` has side effect